README.md

```diff
@@ -1,35 +1,123 @@
-# Membrane Template Plugin
+# Membrane TCP plugin
 
-[![Hex.pm](https://img.shields.io/hexpm/v/membrane_template_plugin.svg)](https://hex.pm/packages/membrane_template_plugin)
-[![API Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_template_plugin)
-[![CircleCI](https://circleci.com/gh/membraneframework/membrane_template_plugin.svg?style=svg)](https://circleci.com/gh/membraneframework/membrane_template_plugin)
+[![Hex.pm](https://img.shields.io/hexpm/v/membrane_tcp_plugin.svg)](https://hex.pm/packages/membrane_tcp_plugin)
+[![API Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_tcp_plugin/)
+[![CircleCI](https://circleci.com/gh/membraneframework/membrane_tcp_plugin.svg?style=svg)](https://circleci.com/gh/membraneframework/membrane_tcp_plugin)
 
-This repository contains a template for new plugins.
+This package provides TCP Source and Sink, that read and write to TCP sockets.
 
-Check out different branches for other flavors of this template.
+## Installation
 
-It's a part of the [Membrane Framework](https://membrane.stream).
+Add the following line to your `deps` in `mix.exs`. Run `mix deps.get`.
 
-## Installation
+```elixir
+	{:membrane_tcp_plugin, "~> 0.1.0"}
+```
+
+## Usage example
 
-The package can be installed by adding `membrane_template_plugin` to your list of dependencies in `mix.exs`:
+The example below shows 2 pipelines. `TCPDemo.Send` downloads an example file over HTTP and
+sends it over TCP socket via localhost:
 
 ```elixir
-def deps do
-  [
-    {:membrane_template_plugin, "~> 0.1.0"}
-  ]
+defmodule TCPDemo.Send do
+  use Membrane.Pipeline
+
+  alias Membrane.{Hackney, TCP}
+  alias Membrane.Pipeline.Spec
+
+  def handle_init(_) do
+    children = [
+      source: %Hackney.Source{
+        location: "https://membraneframework.github.io/static/video-samples/test-video.h264"
+      },
+      tcp: %TCP.Sink{
+        destination_address: {127, 0, 0, 1},
+        destination_port_no: 5001,
+        local_address: {127, 0, 0, 1}
+      }
+    ]
+
+    links = %{
+      {:source, :output} => {:tcp, :input}
+    }
+
+    spec = %Spec{children: children, links: links}
```

Comment:
Why do you use children specification API from Core v(-0.3)? :D

Reply:
I totally forgot about readme, it's not accurate, since it's just the readme of UDP plugin, but with UDP replaced with TCP. But I think I'll prioritize the functionality for now, and replace readme with TODO